module/VuFind/src/VuFind/Db/Entity/Session.php

```diff
@@ -102,4 +103,86 @@ class Session implements EntityInterface
      * )
      */
     protected $created = '2000-01-01 00:00:00';
+
+    /**
+     * Id getter
+     *
+     * @return int
+     */
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    /**
+     * Session Id setter
+     *
+     * @param ?string $sid Session Id.
+     *
+     * @return Session
+     */
+    public function setSessionId(?string $sid): Session
```

Comment:
Looks like this is `string|null` in the auto-generated properties above. Maybe it's worth doing a separate PR to just replace `string|null` with `?string` in all the entity classes for consistency.